### ISSUE TITLE
Passing and retrieving intents in Main Activity Console Error

### ISSUE THREAD

**Issue by NigelKazembe** [chaquo/chaquopy#738]
Good day, 

I am currently facing this issue when trying to retrieve an intent and its extras in the Main Activity of the console demo app. Each time I try to access it, it returns a NullPointerException, is there any way to bypass this or rather solve this issue, or is it impossible to retrieve? Or is there a specific way for doing so in the Main Activity
![image](https://user-images.githubusercontent.com/52897714/199513075-3bb38253-4479-442e-b1f7-9d4f59496e52.png)

The reason is I would like to use an EditTextView to obtain the module name from the user, and then pass it into the run method such that it will match the name entered by the user and run that specific file.

**Comment by mhsmith** [comment]
That's not the correct way to retrieve an intent. You'll should do it in one of the Activity lifecycle methods like onCreate or onResume. Please search for examples of that on developer.android.com, StackOverflow or elsewhere.

**Comment by NigelKazembe** [comment]
> That's not the correct way to retrieve an intent. You'll should do it in one of the Activity lifecycle methods like onCreate or onResume. Please search for examples of that on developer.android.com, StackOverflow or elsewhere.

Actually, that was the first thing that I was doing, used onCreate and onResume, the snippet I sent was because when trying to use the methods where I'm supposed to retrieve it were not working at all.

**Comment by NigelKazembe** [comment]
![image](https://user-images.githubusercontent.com/52897714/199849263-225e90e2-aba7-45f8-893e-1f6ea970ca77.png)
So I finally managed to resolve the issue, previously I tried to retrieve the intent extras in all of the lifecycle methods and the program would return a NullPointerException when trying to set the instance or even static variable in them. SO, I realized that I needed to set the value manually to the instance variable/static variable using a method, and then access the static variable from the static class Task. Weird behavior since I don't normally have to do things this way when working with intents but it worked nonetheless.
